src/app/UploadPilets.tsx

```diff
@@ -1,23 +1,31 @@
 import { FC, ChangeEvent, useState } from 'react';
-import { CustomInput, InputGroup, InputGroupAddon, Button } from 'reactstrap';
+import { CustomInput, InputGroup, InputGroupAddon, Button, Progress, UncontrolledAlert } from 'reactstrap';
 import { jsx } from '@emotion/core';
 import { injectPiletsFromUrl } from './utils';
 
-export interface UploadPiletsProps {}
+export interface UploadPiletsProps { }
 
 export const UploadPilets: FC<UploadPiletsProps> = () => {
   const [file, setFile] = useState({
     value: undefined,
     key: 0,
   });
 
+  const [progress, setProgress] = useState(0);
+  const [failedUpload, setFailedUpload] = useState(false);
+  const [piletName, setPiletName] = useState();
+  const [succeededUpload, setSucceededUpload] = useState(false);
+  const [err, setErr] = useState();
```

Comment:
The current upload input only accepts one upload right now. 

Maybe we change it to multiple?

Reply:
That's not what I mean, but yes, it could also be added there.

What I meant is that you can use the form, start the upload, and then immediately select a new file and start a new upload. So you got two uploads running in parallel (even though the upload either has to take long or one has to be really quick to achieve that).

But even if that scenario that I described is not active we have a problem with the current data model. If I start a new upload I lose the information from the previous one (success? failure?). One overrides the other...